src/tabs/tabs.scss

```diff
@@ -4,100 +4,139 @@
-        background-color $iui-speed-fast ease-out;
-
-      &::after {
-        content: '';
-        position: absolute;
-        z-index: 1;
-        transition: all $iui-speed-fast;
-        height: $iui-xxs;
-        opacity: 0;
-
+  
+      .iui-tab-icon {
         @include themed {
-          background-color: t(iui-color-foreground-primary);
+          fill: t(iui-icons-color-primary);
         }
       }
+    }
 
-      &:hover {
-        text-decoration: none;
+    &[disabled] {
+      cursor: not-allowed;
+      @include themed {
+        color: t(iui-text-color-muted);
+      }
 
+      .iui-tab-icon {
         @include themed {
-          color: t(iui-color-foreground-primary);
-          fill: t(iui-icons-color-primary);
+          fill: t(iui-icons-color-actionable-disabled);
         }
-        transition: color $iui-speed-fast ease-out,
-          background-color $iui-speed-fast ease-out;
       }
     }
+  }
+
+  .iui-tab-icon {
+    width: $iui-icons-default;
+    height: $iui-icons-default;
+    transition: fill $iui-speed-fast ease-out;
+    @include themed {
+      fill: t(iui-icons-color-actionable);
+    }
 
-    &.iui-tabs-active {
-      > a {
-        font-weight: $iui-font-weight-bold;
-        cursor: default;
+    + .iui-tab-label {
+      margin-left: $iui-s;
+    }
+  }
 
-        &:hover {
-          @include themed {
-            color: t(iui-text-color);
-          }
-        }
+  .iui-tab-label {
+    text-align: left;
+  }
 
-        &::after {
-          @include themed {
-            opacity: t(iui-opacity-1);
-          }
-        }
-      }
+  .iui-tab-description {
+    display: none;
+    font-size: $iui-font-size-small;
+    @include themed {
+      opacity: t(iui-opacity-3);
     }
   }
 
-  &.iui-green > li {
-    > a {
-      &:hover {
-        @include themed {
-          color: t(iui-color-foreground-positive);
-          fill: t(iui-icons-color-positive);
-        }
+  &.iui-green .iui-tab {
+    @include iui-focus(iui-focus-positive-box-shadow);
+
+    &.iui-active {
+      @include themed {
+        color: t(iui-color-foreground-positive);
       }
 
-      &::after {
+      .iui-tab-icon {
         @include themed {
-          background-color: t(iui-color-foreground-positive);
+          fill: t(iui-icons-color-positive);
         }
       }
     }
+  }
 
-    &.iui-tabs-active > a:hover {
+  ~ .iui-tabs-content {
+    padding-top: $iui-baseline;
+    padding-bottom: $iui-baseline;
+    box-sizing: border-box;
+  }
+}
+
+@mixin iui-tab-not-animated {
+  .iui-tab {
+    transition: box-shadow $iui-speed-fast ease-out;
+    @include themed {
+      box-shadow: inset 0 0 0 0;
+    }
+    
+    &.iui-active {
       @include themed {
-        color: t(iui-text-color);
+        box-shadow: inset 0 (-$iui-xxs) 0 0;
       }
     }
   }
+
+  .iui-tab-stripe {
+    display: none;
+  }
+}
+
+@mixin iui-tab-animated {
+  ~ .iui-tab-stripe {
+    width: 100%;
```

Comment:
If we are setting `width` and `left` dynamically, then I don't think we need to have rules for them in CSS.

Reply:
Removed